WalletWasabi/BitcoinCore/RpcParser.cs

```diff
@@ -8,8 +9,34 @@
 
 namespace WalletWasabi.BitcoinCore
 {
-	public class RpcParser
+	public static class RpcParser
 	{
+		public const string SpentError1 = "bad-txns-inputs-missingorspent";
```

Comment:
RpcParser doesn't seem to be the correct place for these texts. Can you move it back to ExceptionExtensions?

Reply:
I took the code out into a new file, called `RpcErrorTools` instead. The .NET Exception concept should not really be aware of the Bitcoin Core error messages.